packages/product-service/addProduct/addProduct.ts

```diff
@@ -0,0 +1,67 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { Client } from 'pg';
+import { Product } from '../models';
+import { cors } from '../cors';
+
+export const addProduct: APIGatewayProxyHandler = async (event) => {
+  console.log(`Request: ${JSON.stringify(event, null, 2)}`);
+
+  if (!event.body) {
+    return {
+      statusCode: 400,
+      body: 'Bad request',
+      headers: cors,
+    };
+  }
+
+  const client = new Client();
+
+  try {
+    await client.connect();
```

Comment:
@dkovalishyn It's better to encapsulate such repetitive logic as db connections/error handling/response processing/DB transaction handling in a single place (DRY principle). This way, you won't have to change code in every file, when you want to modify response handling, for example

It's a wish for the future, no need to implement it right now - it won't affect your score.